src/types.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+export const AppTypes = PropTypes.shape({
+  isClockVisible: PropTypes.bool.isRequired,
+  clockName: PropTypes.string.isRequired,
+});
+
+export const ClockTypes = PropTypes.shape({
+  date: PropTypes.object,
+  timer: PropTypes.func,
+});
```

Comment:
Clock accepts only 1 prop - Name. You can describe it in the file with your clock component. No need to make this file. Don't mix state and props

Reply:
ooo, now I understood))))